datasets/numeric_fused_head/README.md

```diff
@@ -0,0 +1,190 @@
+---
+annotations_creators:
+  identification:
+  - expert-generated
```

Comment:
The identification dataset actually has two with different quality annotation types:
1) the training and development splits were created automatically, using a constituency parser.
2) the test set was curated by experts (the authors)

Reply:
Added!